web/client/plugins/Widgets.jsx

```diff
@@ -37,13 +53,17 @@ compose(
             getMaximizedState,
             dependenciesSelector,
             (state) => mapLayoutValuesSelector(state, { right: true}),
-            (id, widgets, layouts, maximized, dependencies, mapLayout) => ({
+            state => state.browser && state.browser.mobile,
```

Comment:
I think we have already an isMobile selector anyway I'm wondering if we should rely only on the mobile selector, we already know that if a plugin is listed in the plugins.mobile list of localConfig this will be available only on mobile device. We could use a cfg only on mobile to enable the single widget visualization.

Reply:
This is an option, but even in this case we need to distinguish between Mobiles where viewport is usually below 600px (400+px actually) and tablets where it's more than 600px (checked here https://screensiz.es/tablet). For tablets it makes sense to have several columns because there are more space to utilize by widgets, especially in landscape mode